### ISSUE TITLE
Sparse tensor for label

### ISSUE THREAD

**Issue by MisayaZ** [bai-shang/OCR_TF_CRNN_CTC#4]
the label is a sparse_placeholder, however, the lable value which you feed into the placeholder is not a sparse matrix. Is it wrong?

**Comment by bai-shang** [comment]
I used sparse_placeholder since the first parameter of tf.nn.ctc_loss need int32 SparseTensor.
During feeding dict, tensorflow can automatically convert dense tensor to sparse tensor correctly(but this will cost big memory).
